src/OrchardCore/OrchardCore/Modules/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -312,7 +312,7 @@ private static void AddAntiForgery(OrchardCoreBuilder builder)
                 var settings = serviceProvider.GetRequiredService<ShellSettings>();
                 var environment = serviceProvider.GetRequiredService<IHostEnvironment>();
 
-                var cookieName = "orchantiforgery_" + HttpUtility.UrlEncode(settings.Name + environment.ContentRootPath);
+                var cookieName = "orchantiforgery_" + HttpUtility.UrlEncode(settings.Name + environment.ContentRootPath.Replace("(", "").Replace(")", ""));
```

Comment:
Can we define a `static` array of the invalid characters, then we can remove them from `ContentRootPath`

Reply:
Let's use the slugify method here.